components/ledger/internal/gen/wire_gen.go

```diff
@@ -62,36 +62,39 @@ func InitializeService() *service.Service {
 		InstrumentRepo:   instrumentPostgreSQLRepository,
 		MetadataRepo:     metadataMongoDBRepository,
 	}
-	accountHandler := &ports.AccountHandler{
+	accountHandler := &http.AccountHandler{
 		Command: useCase,
 		Query:   queryUseCase,
 	}
-	portfolioHandler := &ports.PortfolioHandler{
+	portfolioHandler := &http.PortfolioHandler{
 		Command: useCase,
 		Query:   queryUseCase,
 	}
-	ledgerHandler := &ports.LedgerHandler{
+	ledgerHandler := &http.LedgerHandler{
 		Command: useCase,
 		Query:   queryUseCase,
 	}
-	instrumentHandler := &ports.InstrumentHandler{
+	instrumentHandler := &http.InstrumentHandler{
 		Command: useCase,
 		Query:   queryUseCase,
 	}
-	organizationHandler := &ports.OrganizationHandler{
+	organizationHandler := &http.OrganizationHandler{
 		Command: useCase,
 		Query:   queryUseCase,
 	}
-	productHandler := &ports.ProductHandler{
+	productHandler := &http.ProductHandler{
 		Command: useCase,
 		Query:   queryUseCase,
 	}
 	app := http.NewRouter(accountHandler, portfolioHandler, ledgerHandler, instrumentHandler, organizationHandler, productHandler)
 	logger := mzap.InitializeLogger()
 	server := service.NewServer(config, app, logger)
+	grpcServer := grpc.NewRouterGRPC(useCase, queryUseCase)
```

Comment:
It could be clearer keep this var as: 

```suggestion
	grpcRouter := grpc.NewRouterGRPC(useCase, queryUseCase)
```
There is already a var with similar name on the line below